graphql_api/types/plan_representation/plan_representation.graphql

```diff
@@ -1,12 +1,8 @@
 type PlanRepresentation {
-  marketingName: String!
-  planName: String!
-    @deprecated(
-      reason: "Plan representations have used `value` for a while, making the frontend code hard to change"
-    )
-  value: String!
-  billingRate: String
   baseUnitPrice: Int!
```

Comment:
sorting these in ascending order for easier parsing

Reply:
doing the lords work 🙏